docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -609,7 +609,28 @@ TABLE AS </sql/create-table-as>` syntax.
 
 The connector supports the following [](/sql/alter-table) statements.
 
+(delta-lake-create-or-replace)=
+
```

Comment:
remove empty line

Reply:
All of the other anchors on the page have a space between the them and the section header.

Reply:
I know .. we want to get rid of them all eventually